packages/flutter/lib/src/widgets/text.dart

```diff
@@ -1401,3 +1521,21 @@ class _SelectableTextContainerDelegate extends MultiSelectableSelectionContainer
     super.didChangeSelectables();
   }
 }
+
+class _TextSpanContentRange extends SelectedContentRange<TextSpan> {
+  _TextSpanContentRange({
+    this.start = -1,
+    this.end = -1,
```

Comment:
Why the redundant parameters? Could you maybe make them private at least so their aren't two getters for the same value?